index.html

```diff
@@ -1,58 +1,147 @@
+			margin: 0px 10px;
+			padding: 10px;
+			border-radius: 4px;
+		}
+		
+		.moreinfo {
+			width: 1010px;
+			height: 70px;
+			color: white;
+			font-weight: lighter;
+			line-height: 1.5em;
+			background-color: rgba(0, 0, 0, 0.6);
+			margin-top: 10px;
+			padding: 10px;
+			border-radius: 4px;
+		}
+		
+		.moreinfo a {
+			color: white;
+		}
+		
+		#container {
+			display: flex;
+			flex-flow: wrap;
+			flex-wrap: wrap;
+			align-content: center;
+			justify-content: center;
+			flex-direction: row;
+			margin: 0 !important;
+			flex-basis: auto;
+		}
+		
+		#question {
+			width: 100px;
+			height: 30px;
+			color: white;
+			font-weight: lighter;
+			margin-bottom: 10px;
+			background-color: rgba(0, 0, 0, 0.6);
+			border-radius: 3px;
+			display: flex;
+			align-content: center;
+			justify-content: center;
+			align-items: center;
+		}
+		
+		#question:hover {
+			cursor: pointer;
+		}
+		
+		#infobox {
+			display: none;
+		}
+		
+		img {
+			width: 100%;
+			margin: 20px;
+		}
+		
+		h1 {
+			font-size: 3em;
+			color: white;
+			padding-top: 80px;
+			/*background-color: rgba(183, 58, 98, 0.89);*/
+		}
 	</style>
 </head>
 
 <body>
 	<!--<img style ="width: 80px; height: 80px; margin-top:10px" src ="./images/logo.png">-->
-    <h1 >The Coding Space Projects</h1>
-   <div id="container">
-	<div>
-		<a href="./scratch"><img style ="width:300px; height: 300px" id="banner1" src="./images/scratch2.png" onmouseover="this.src='./images/scratchhover.png'" onmouseout="this.src='./images/scratch2.png'" /></a>
-	</div>
-	<div>
-		<a href="http://woofjs.com/create"><img  style ="width:300px; height: 300px" id="banner3" src="./images/woof2.png" onmouseover="this.src='./images/woofhover.png'" onmouseout="this.src='./images/woof2.png'"/></a>
+	<h1>The Coding Space Projects</h1>
+	<div id="container">
+		<div>
+			<a href="./scratch"><img style="width:300px; height: 300px" id="banner1" src="./images/scratch2.png" onmouseover="this.src='./images/scratchhover.png'" onmouseout="this.src='./images/scratch2.png'" /></a>
+		</div>
+		<div>
+			<a href="http://woofjs.com/create"><img style="width:300px; height: 300px" id="banner3" src="./images/woof2.png" onmouseover="this.src='./images/woofhover.png'" onmouseout="this.src='./images/woof2.png'" /></a>
+		</div>
+		<div>
+			<a href="./web"><img style="width:300px; height: 300px" id="banner2" src="./images/web2.png" onmouseover="this.src='./images/webhover.png'" onmouseout="this.src='./images/web2.png'" /></a>
+		</div>
+		<div class="infoparts">
+			<div id="question">Learn More</div>
+			
+		</div>
+		<div id="infobox" class="infoparts">
+			<div class="textbox">Scratch is a block-based coding platform that lets you code games and animations without worrying about typing. All you need to do is drag and drop!</div>
+			<div class="textbox">WoofJS is a JavaScript framework that helps you transition from block-based coding in Scratch to syntax-based coding in JavaScript</div>
```

Comment:
I would focus more on the things students care about:

- create animations and games
- can be used on computers, tablets and phones
- JavaScript (you might want to explain what JS is if you can in short way that makes sense for kids)

...and less on the pedagogical benefits of it as a transition tool